### PR TITLE
Deprecate reaction questions

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

File: molo/core/models.py [hunk 1]
```diff
@@ -1157,32 +1055,35 @@ def get_site(self):
...
+
+        if 'footers-%s' % slug not in descendants:
             footer_index = FooterIndexPage(
                 title='Footers', slug=('footers-%s' % (
-                    generate_slug(self.title), )))
+                    slug, )))
             self.add_child(instance=footer_index)
             footer_index.save_revision().publish()
+
+        if 'footers-%s' % slug not in descendants:
```
Comment (KaitCrawford) [inline molo/core/models.py]: Copy paste error? I'm guessing this is meant to be `if 'tags-%s'...` ?

File: molo/core/wagtail_hooks.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 from django.conf.urls import url
 
 from molo.core.admin import (
-    ReactionQuestionsModelAdmin, ReactionQuestionsSummaryModelAdmin,
+    # ReactionQuestionsModelAdmin, ReactionQuestionsSummaryModelAdmin,
```
Comment (KaitCrawford) [inline molo/core/wagtail_hooks.py]: Why is this line commented out rather than deleted?

File: molo/core/wagtail_hooks.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
     AdminViewGroup
 )
-from molo.core.admin_views import ReactionQuestionResultsAdminView, \
-    ReactionQuestionSummaryAdminView
+# from molo.core.admin_views import ReactionQuestionResultsAdminView, \
+#     ReactionQuestionSummaryAdminView
```
Comment (KaitCrawford) [inline molo/core/wagtail_hooks.py]: Same question about commented lines here

**Review by KaitCrawford — COMMENTED** [review]
@moh-moola Very small comments.

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Comment by codiebeulaine** [comment]
I think migrations are necessary here? When I run `makemigrations` it creates a migration that deleted reaction questions, a failing migration

**Comment by moh-moola** [comment]
@codiebeulaine totally overlooked, will patch and make a release

**Comment by codiebeulaine** [comment]
cool cool. Can I get an ETA on that release?

**Pull request merged**